### PR TITLE
Add platforms

### PR DESCRIPTION
**Pull request by juanmanuelramallo**
Add platform pages for each connector page.

image

### CODE REVIEW

File: _plugins/to_platforms.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module Jekyll
+  class ToPlatforms < Jekyll::Generator
+    def generate(site)
```
Comment (matiascondem) [inline _plugins/to_platforms.rb:3]: would be nice to document what a Site is right?

File: _plugins/to_platforms.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      platforms = site.data['platforms']
```
Comment (matiascondem) [inline _plugins/to_platforms.rb:4]: super nitpicky but why single quotes instead of double quotes?

**Review by matiascondem — APPROVED** [review]

File: _plugins/to_platforms.rb (hunk 1 above)
Comment (juanmanuelramallo) [reply _plugins/to_platforms.rb:3]: It's just how Jekyll plugins work. documentation is here https://jekyllrb.com/docs/plugins/ https://jekyllrb.com/docs/plugins/generators/

File: _plugins/to_platforms.rb (hunk 2 above)
Comment (juanmanuelramallo) [reply _plugins/to_platforms.rb:4]: They are the sam(ish) tbh, no rubocop rules in here hehe

**Pull request merged**